### PR TITLE
[new release] pgx_value_ptime, pgx_value_core, pgx_unix, pgx_lwt_unix, pgx_lwt_mirage, pgx_lwt, pgx_async and pgx (2.2)

### PR DESCRIPTION
**Pull request by gtrak**
Ocaml postgresql client

- Project page: <a href="https://github.com/arenadotio/pgx">https://github.com/arenadotio/pgx</a>
- Documentation: <a href="https://arenadotio.github.io/pgx">https://arenadotio.github.io/pgx</a>

##### CHANGES:
Added support for ocaml 5.0 deprecations https://github.com/arenadotio/pgx/pull/129

### CODE REVIEW

File: packages/pgx/pgx.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "Pure-OCaml PostgreSQL client library"
+description:
+  "PGX is a pure-OCaml PostgreSQL client library, supporting Async, LWT, or synchronous operations."
+maintainer: ["Arena Developers <silver-snakes@arena.io>"]
+authors: ["Arena Developers <silver-snakes@arena.io>"]
+license: "LGPL-2.0-only with OCaml-LGPL-linking-exception"
```
Comment (raphael-proust) [inline packages/pgx/pgx.2.2/opam:7]:
```suggestion
license: "LGPL-2.0-only WITH OCaml-LGPL-linking-exception"
```

And same for all other packages. This follows the standard notation of licenses.

File: packages/pgx_unix/pgx_unix.2.2/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "PGX using the standard library's Unix module for IO (synchronous)"
+description:
+  "PGX using the standard library's Unix module for IO (synchronous)"
+maintainer: ["Arena Developers <silver-snakes@arena.io>"]
+authors: ["Arena Developers <silver-snakes@arena.io>"]
+license: "LGPL-2.0-only with OCaml-LGPL-linking-exception"
```
Comment (raphael-proust) [inline packages/pgx_unix/pgx_unix.2.2/opam:7]:
```suggestion
license: "LGPL-2.0-only WITH OCaml-LGPL-linking-exception"
```

File: packages/pgx_value_ptime/pgx_value_ptime.2.2/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Pgx_value converters for Ptime types"
+description: "Pgx_value converters for Ptime types"
+maintainer: ["Arena Developers <silver-snakes@arena.io>"]
+authors: ["Arena Developers <silver-snakes@arena.io>"]
+license: "LGPL-2.0-only with OCaml-LGPL-linking-exception"
```
Comment (raphael-proust) [inline packages/pgx_value_ptime/pgx_value_ptime.2.2/opam:6]:
```suggestion
license: "LGPL-2.0-only WITH OCaml-LGPL-linking-exception"
```

**Review by raphael-proust — CHANGES_REQUESTED** [review]

**Comment by raphael-proust** [comment]
@gtrak Can you also apply those changes to your upstream repo so they persist for the next release?

**Comment by mseri** [comment]
Thanks

**Comment by gtrak** [comment]
thanks @mseri I'll update the licenses on our end for next time

**Pull request merged**